### PR TITLE
New Package: lutris-0.3.6.3

### PR DESCRIPTION
**Pull request by JayVii**
#1686 
Only tested it on a 32-bit machine. Shouldn't be any different for 64, though.

Contributing for the first time, anything I have to change with the template?

------
setup.py-script provided by lutris didn't work (especially not with repackaging).

### CODE REVIEW

File: srcpkgs/lutris/template [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+homepage="https://lutris.net/"
+license="GPL-3"
+distfiles="http://lutris.net/releases/lutris_${version}.tar.gz"
+checksum=a9061403ef8844b2c1443b1c5f15fe98af224d652d66268b47ae8e215e289058
+depends="python python-yaml gir-freedesktop python-xdg glib-networking"
+
+do_extract() { 
+	mkdir -p ${DESTDIR}
+	tar xzpvf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/lutris_${version}.tar.gz -C ${DESTDIR}
+}
```
Comment (xtraeme) [inline srcpkgs/lutris/template]: This do_extract is really wrong...

**Comment by JayVii** [comment]
What should it be like, instead?

File: srcpkgs/lutris/template (hunk 1 above)
Comment (JayVii) [inline srcpkgs/lutris/template]: should the "tar" be in "do_install()"?  (Saw that on the brother-dcp197c-cupswrapper package)

**Comment by JayVii** [comment]
Wow, finally. This has been a lot harder than it should have been :s 
Anyway, made it 32/64 bit only now, as all games on lutris are x86/x86_64 anyway.

**Comment by JayVii** [comment]
holy.... -.-

**Comment by pullmoll** [comment]
I feel like giving some hints on how to create pull requests. I had to learn this myself and it can be confusing at first. I don't want to criticize, just help, if that is possible :-)

+ In your fork of void-packages create a new branch like `git checkout -b <somename>`
+ Set the upstream `git remote add upstream git://github.com/voidlinux/void-packages.git`
+ A helpful script to publish (and later de-publish) your branch `<somename>` can be [found here](http://repo.voidlinux.de/tools/mybranch).
    + Put it in e.g. you $(HOME)/bin and make sure it is executable and $(HOME)/bin is in your path
+ Run `mybranch <somename>` to publish the branch and set some git config entries
    + You can take a look at all your fork's branches with `git branch -a`
+ Edit your files, `git add` and `git commit` as you like
+ You may also `git push` to to save your local work (without polluting master)
+ Once you are happy with your changes, and if there are lots of commits, use this procedure:
    + `git log` to find the commit `id`before your first change (the first 7 digits should suffice)
    + `git rebase -i <id>` and change subsequent commits from "keep" to sqash "s"
    + edit the commit message you want to use for it all
+ Now, if some time is gone since you started your branch, ideally you should rebase it to upstream master with `git pull --rebase upstream master`.
+ Finally you can now `git push` or perhaps, if you pushed before and rebased, `git push --force`
+ On `https://github.com/JayVii/void-packages` now create the pull request (possible after selecting the branch)
+ Once the pull request is accepted (or denied) you can de-publish the branch with `mybranch -d <somename>`
+ Afterwards you can also delete your local branch with `git branch -d <somename>` or, if there are pending changes you want to discard, `git branch -D <somename>`.

This is my procedure and I think it's not too difficult once you got used to it.
I wrote this without testing in parallel, so be careful. At least some details may be useful info for you.

Cheers!

**Comment by JayVii** [comment]
Thank you very much @pullmoll :)
I'll save this as so i can check on it again next time.

I felt bad spamming out mini-commits, but being new to this I simply don't know better :-1: Anyway, as soon as my 2 current commits are out of the way (accepted or rejected), i'll re-clone the git and set up proper branches.

**Comment by beefcurtains** [comment]
No need to reclone, just ``git rebase -i $parent_commit``

**Pull request closed without merging**